tests/test_connector_slack.py

```diff
@@ -145,6 +145,7 @@ class TestConnectorSlackAsync(asynctest.TestCase):
 
         connector.opsdroid.parse.reset_mock()
         connector.lookup_username.side_effect = ValueError
+        connector.lookup_username.side_effect = KeyError
```

Comment:
This has changed the side effect from `ValueError` to `KeyError` and so no longer tests `ValueError`.

I suggest you copy that whole block from L146 to L150 and just switch out the exception.

Reply:
Sorry, it was a silly mistake. Making the changes now